### ISSUE TITLE
TTN MQTT broker appears to be broken.

### ISSUE THREAD

**Issue by whmountains** [TheThingsNetwork/ttn#732]
This is a **bug report** for **the backend**.

**Tl;Dr:** I'm unable to view uplink or send downlink packets from the web console, ttnctl, or the Node.js SDK.  The last-seen counter is working, but only updates when I refresh the page, leading me to believe that there is a problem with the part of the TTN stack that sends out live data.  My best guess is the MQTT broker.

> What do you want to do?

I want to send downlink and receive uplink data from the console and Node.js SDK.

> What steps did you take?

* Log in to TTN.
* Select "Console"
* Click "Applications"
* Select my application.
* Switch to the "Devices" tab.
* Choose my device.
* Switch to the "Data" tab.
* Set the filters to show all types of packets.

I also tried viewing packets via 

> What went wrong or what is missing?

No packets are visible, though the logs from my LoRa gateway show that it's successfully forwarding packets.  The last-seen field only updates when I refresh the page.  (e.g. right now it's at 6 minutes, but when I reload it will jump back to 2 seconds and count down from there.)

I also tried the ttn cli but nothing shows there either.  I can even manually schedule a downlink from the TTN console but still nothing shows up on the data view.

It was working fine earlier, then suddenly stopped.  What I did just before it stopped:

* I modified some unrelated application code on my Murata LoRa module.  I reverted my changes but TTN still seems not to be working.

* I selected "reset frame counters" but clicked cancel after I read the scary message.  Maybe that messed everything up?

> Do you have Screenshots?

Sure do!

**Status & Frame Counters**

After leaving the page open for 8 minutes:
![screen shot 2018-07-26 at 8 32 41 am](https://user-images.githubusercontent.com/8680147/43265508-8a667c6e-90ae-11e8-8074-a91af6d2c1b8.png)

After a reload:
![screen shot 2018-07-26 at 8 33 48 am](https://user-images.githubusercontent.com/8680147/43265539-a524f04e-90ae-11e8-8119-a6bf8944f514.png)

**Data view**

For a specific device:
![screen shot 2018-07-26 at 8 34 42 am](https://user-images.githubusercontent.com/8680147/43265585-c882926c-90ae-11e8-9b38-c7d494038825.png)

For the entire application:
![screen shot 2018-07-26 at 8 35 38 am](https://user-images.githubusercontent.com/8680147/43265616-e502bce6-90ae-11e8-8ad1-829c4c42d855.png)

For the gateway:
![screen shot 2018-07-26 at 8 36 22 am](https://user-images.githubusercontent.com/8680147/43265650-00724a6e-90af-11e8-8d1b-9b9567725038.png)

**What my gateway shows:**

![screen shot 2018-07-26 at 8 38 03 am](https://user-images.githubusercontent.com/8680147/43265735-3d2dd392-90af-11e8-9239-89f587c65575.png)
![screen shot 2018-07-26 at 8 37 41 am](https://user-images.githubusercontent.com/8680147/43265737-408e1934-90af-11e8-9467-ce2bbd17fd53.png)
![screen shot 2018-07-26 at 8 38 57 am](https://user-images.githubusercontent.com/8680147/43265796-67b4e48e-90af-11e8-957e-09905ff71199.png)

Serial output from my device:

![screen shot 2018-07-26 at 8 44 25 am](https://user-images.githubusercontent.com/8680147/43266119-221e301e-90b0-11e8-91fc-a70692640d19.png)

> What kind of OS/Browser/Gateway are you using? Which version?

LoRa Module: Murata CMWX1ZZABZ-078
Dev Board: Tlera Grasshopper
Gateway: Laird Sentrius RG191
My computer: MacOS 10.13.5
Browser: Firefox 63, but I also tried Chromium 61, the ttn cli, and the Node.js SDK with identical results.

> What are the IDs and EUIs of your Device/Gateway? (if applicable)

Gateway ID: au-test-1
Device ID: au-aio-valves-revc-1
Device EUI: 000964CE3AE1AF8E

> What do your configuration files look like?

**Gateway Configuration**
![screen shot 2018-07-26 at 8 48 03 am](https://user-images.githubusercontent.com/8680147/43266280-a294526e-90b0-11e8-9b5a-610cc5e97c8b.png)
![screen shot 2018-07-26 at 8 48 24 am](https://user-images.githubusercontent.com/8680147/43266296-aef3f514-90b0-11e8-808a-2b8e9dac6210.png)

The keys are blanked out, of course, but I double-checked them.

**Device Configuration**

This is the exact code I'm using: (with the appKey removed, of course)

```cpp
#include "LoRaWAN.h"

const char *appEui  = "70B3D57ED0010D89";
const char *appKey  = "REDACTED";
const char *devEui  = "000964CE3AE1AF8E";

void setup( void )
{
    Serial.begin(9600);
    
    LoRaWAN.begin(US915);

    LoRaWAN.setSubBand(2);
    LoRaWAN.joinOTAA(appEui, appKey, devEui);

    Serial.println("JOIN( )");
}

void loop( void )
{
    if (LoRaWAN.joined() && !LoRaWAN.busy())
    {
        Serial.print("TRANSMIT( ");
        Serial.print("TimeOnAir: ");
        Serial.print(LoRaWAN.getTimeOnAir());
        Serial.print(", NextTxTime: ");
        Serial.print(LoRaWAN.getNextTxTime());
        Serial.print(", MaxPayloadSize: ");
        Serial.print(LoRaWAN.getMaxPayloadSize());
        Serial.print(", DR: ");
        Serial.print(LoRaWAN.getDataRate());
        Serial.print(", TxPower: ");
        Serial.print(LoRaWAN.getTxPower(), 1);
        Serial.print("dbm, UpLinkCounter: ");
        Serial.print(LoRaWAN.getUpLinkCounter());
        Serial.print(", DownLinkCounter: ");
        Serial.print(LoRaWAN.getDownLinkCounter());
        Serial.println(" )");

        LoRaWAN.beginPacket();
        LoRaWAN.write(0xef);
        LoRaWAN.write(0xbe);
        LoRaWAN.write(0xad);
        LoRaWAN.write(0xde);
        LoRaWAN.endPacket();
    }

    delay(1000);
}
```

> What do your log files look like?

See the screenshots.

> Can you fix this yourself and submit a pull request?

Probably not.

**Comment by htdvisser** [comment]
This is not going to get fixed in V2. The Things Stack (V3) should not have this problem.
